daattali/timevis
# Is able to past list-cols to timevis (#11)

Only two months later I took the 10 minutes it takes to checkout your last commit, choose a clean working tree and squash my commits.... Lmao sorry @daattali 


<details><summary>A bunch of meaningless commits that will be squashed</summary>

* Added glue to suggests

* Changed encoding function to produce json objects

* Added helper for more informative test output

* Added calls to info output helper

* Only writes list-cols as json

Passes regression tests again

* Added unit test to test nested col behaviour

(Failing)

* new test now passing

* Now passes list-cols (such as nestedGroups) through

* Update version number

* Added .DS_Store to gitignore

* Added newest release of vis.js

* Update timevis.yaml to point at vis-4.21

* Added .DS_Store to .gitignore

* Updated unit test

* commented info comp test helper

* Added unit test to check length-1 listing

* Added list check to dataframetod3

* added null handle to dftod3

* Commented null handle out in dataframetoD3

Mainly because I should probably write more regression tests for it before blindly putting it in
I should probably implement a check on the whole column, rather than checking in an apply to check if any are vectors in the col

* Added timeline plus

* Updated dependencies to use timeline-plus

* renamed timeline-plus back to vis, setup src properly

* Update timevis.yaml

* Changed references from vis to timeline

* Update timevis.yaml

* Revert "Changed references from vis to timeline"

This reverts commit d7c169e8b869c6b8b04f7a3d9fcd9a210b7d9049.

* changed all refs to timeline to vistime because of collision

* replaced references to vis with timeline

* updated checks in dataframetod3 coercion

added a check for logical

* Replaced timeline.js with my own fixed version

* Extra test case to cover when empty list passed

* dftod3 Passes empty lists and null as NA

* Revert "Merge pull request #7 from mstr3336/upgrade-timeline-plusplus"

This reverts commit 38bd4e47772690dd649071edc855db2e76b1dad2, reversing
changes made to 90501148f2120fec6f5f7f07b54fe815ff956b8f.

</details>
